packages/affine/model/src/elements/brush/brush.ts

```diff
@@ -138,21 +138,19 @@ export class BrushElementModel extends GfxPrimitiveElementModel<BrushProps> {
     instance['_local'].delete('commands');
   })
   @derive((lineWidth: number, instance: Instance) => {
-    const oldBound = instance.elementBound;
```

Comment:
Good catch! This change effectively addresses the issue mentioned, but it could be even better by fixing it with just one line of code.

```ts
    const oldBound = Bound.fromXYWH(instance.deserializedXYWH);
```

Reply:
Done!